sdk/keyvault/azure-security-keyvault-keys/src/delete_key_operation.cpp

```diff
@@ -33,8 +35,8 @@ Azure::Security::KeyVault::Keys::DeleteKeyOperation::PollInternal(Azure::Core::C
         break;
       }
       default:
-        throw KeyVaultException(
-            "Unexpected operation status from Service response.", std::move(rawResponse));
```

Comment:
Same question here about keeping the custom error message, or if it is a better dev experience to leave it out.

Reply:
Better not to add the extra manual label and use only what's coming back from the service